i/instana-agent-operator/instana-agent-operator_rhel8.2.sh

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------
+#
+# Package        : instana-agent-operator
+# Version        : v1.0.4
+# Source repo    : https://github.com/instana/instana-agent-operator.git
+# Tested on      : RHEL 8.2
+# Script License : Apache License, Version 2 or later
+# Maintainer     : Amit Sadaphule <amits2@us.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+set -eu
+
+CWD=`pwd`
+
+if [ "$#" -lt 1 ]; then
+    echo "Usage: ./<script> <AGENT_KEY> [<CLUSTER_NAME>]"
+    echo "where, CLUSTER_NAME is an optional paramater whose default value is kind"
+    exit 1
+fi
+
+AGENT_KEY=$1
+CLUSTER_NAME=${2:-kind}
+
+# Install dependencies
+yum install -y java-11-openjdk-devel wget git
+
+# Download and install maven
+wget https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
+tar xzf apache-maven-3.6.3-bin.tar.gz
+rm -rf apache-maven-3.6.3-bin.tar.gz
+
+# Set enviroment variables
+export MAVEN_HOME=$CWD/apache-maven-3.6.3
+JDK_PATHS=$(compgen -G '/usr/lib/jvm/java-11-openjdk-*')
+export JAVA_HOME=${JDK_PATHS%$'\n'*}
+export PATH=$JAVA_HOME/bin:$MAVEN_HOME/bin:$PATH
+
+# Clone the repo, run build+tests and build image
+git clone https://github.com/instana/instana-agent-operator.git
+cd instana-agent-operator/
+git checkout v1.0.4
```

Comment:
can we accept the version as an input to the script, keeping the default as v1.0.4, if not provided?

Reply:
Updated the script to accommodate this.